pkgs/applications/networking/mailreaders/mutt/default.nix

```diff
@@ -40,6 +40,8 @@ stdenv.mkDerivation rec {
       url = "https://gitlab.com/muttmua/mutt/-/commit/04b06aaa3e0cc0022b9b01dbca2863756ebbf59a.patch";
       sha256 = "117mm757yj4k4cb9f1cmc9p0dqmi2mf92qsxvi8a794b9kdj5m2z";
     })
+    # CVE-2021-3181
+    ./CVE-2021-3181.patch
```

Comment:
Can you use `fetchpatch` instead of pulling the file into nixpkgs? See the line just above.

Reply:
Sure. It is 3 commits though, all 3 referenced by the CVE.